src/CommunityToolkit/Xamarin.CommunityToolkit/Views/Snackbar/Helpers/SnackBarLayout.ios.macos.cs

```diff
@@ -16,11 +14,11 @@ class SnackBarLayout
 
 		public nfloat PaddingBottom { get; set; } = 10.0f;
 
-		public nfloat PaddingLeading { get; set; } = 10.0f;
+		public nfloat PaddingLeft { get; set; } = 10.0f;
```

Comment:
Hm, why Padding **Left**, but Margin **Leading**?

Reply:
Agree, fixed.
Sorry for the delay